products/php.md

```diff
@@ -56,8 +56,8 @@ releases:
   - releaseCycle: "7.0"
     cycleShortHand: "700"
     release: 2015-12-03
-    support: 2018-12-12
-    eol:     2019-01-01
+    support: 2017-12-03
```

Comment:
A better source would be [the code that generates the diagram](https://github.com/php/web-php/blob/382fed715739e4127dc3cd827fb7a819736a43f4/include/branches.inc#L12-L39) where 7.0 is overwritten to `2018-01-04` for active support.

```suggestion
    support: 2018-01-04
```

Reply:
Yeah, that's right. The last 7.0 regular release (7.0.27) was [released on 2018-01-04](https://www.php.net/ChangeLog-7.php#7.0.27).